setup.ml.in

```diff
@@ -88,7 +88,7 @@ let cxx_flags () : unit =
     ~dump:true
     ~short_desc:(fun () -> "c++ flags")
     "cxxflags"
-    (fun () -> "-std=c++11 -I$standard_library") |>
+    (fun () -> "-std=c++11") |>
```

Comment:
looks like that this is some kind of merge error, as it will break everything if compiler is not installed into the system path.